make_BAF_igv.py

```diff
@@ -0,0 +1,60 @@
+#! /usr/bin/env python3
+import sys
+import argparse
+import vcf
+
+def calc_baf(args):
+    output_file = open(args.outputfile, 'w')
+    vcf_file = args.inputfile
+    with open(vcf_file, 'r') as vcf_input_file: 
+        vcf_reader = vcf.Reader(vcf_input_file)
```

Comment:
maybe here a try Except? I assume the vcf module checks if the VCF is actually correctly formatted, but if the user provides a .txt for example? or a modified (incorrectly formatted) VCF? Maybe put an error here, stating that something is incorrect about the VCF input

You can also let argparse check for the .vcf extension (so that .txt is not allowed for example)

Reply:
Included check on 'fileformat' being present in the metadata of the inputfile.
Does not catch all possible errors, but at least  that a VCF file was used.